test/project.xml

```diff
@@ -8,4 +8,7 @@
         <copyright>2005-2015 Laurent Jouanneau and other contributors</copyright>
         <creator name="Laurent Jouanneau" email="laurent@jelix.org" active="true"/>
     </info>
+    <entrypoints>
```

Comment:
this is deprecated in Jelix 1.7 (replaced by the framework.ini file). Remove this addition.